src/index.ts

```diff
@@ -4,6 +4,7 @@ export { Composer } from './composer'
 export { Middleware, MiddlewareFn } from './middleware'
 export { Router } from './router'
 export { TelegramError } from './core/network/error'
+export { default as Telegram } from './telegram'
```

Comment:
Change https://github.com/telegraf/telegraf/blob/38d3479f21dfec2f4c0c03e09ced2286019816d0/src/telegram.ts#L6 to

```ts
export class Telegram extends ApiClient {
```

and then
```suggestion
export { Telegram } from './telegram'
```